static/js/test_constants.js

```diff
@@ -27,148 +27,230 @@ import {
 } from "./constants"
 
 export const ELASTICSEARCH_RESPONSE = deepFreeze({
```

Comment:
There aren't major changes here, just a lot of tiny ones. I copied and adjusted this from a local response to make all the tweaks at once.